PowerShellGet/PSModule.psm1

```diff
@@ -5057,7 +5067,7 @@ function New-ScriptFileInfo
 
         if(-not $Version)
         {
-            $Version = [Version]'1.0'
+            $Version = '1.0'
         }

```

Comment:
Can you explain more about what you mean by validating?  I understood this code is setting the default value of $Version to '1.0'.  What sort of validation are you wanting me to perform?